src/ensemble/basic_ensemble_solve.jl

```diff
@@ -135,13 +135,12 @@ function __solve(prob::AbstractEnsembleProblem,
 
 end
 
-function batch_func(i,prob,alg;kwargs...)
+function batch_func(i,prob,alg,ensemblealg;kwargs...)
```

Comment:
I think we may have to update DiffEqGPU that is using this internal.

Reply:
Would making it a kwarg avoid that?